### ISSUE TITLE
How to update Azure AppService WordPress blog using Project Nami download page

### ISSUE THREAD

**Issue by kppakistan** [ProjectNami/projectnami#289]
Hi,

We have setup an app service in Azure using Project Nami for our Word Press blog. To install new plugins, it is required to update WordPress and it says to update using Project Nami download page. Attached is the screenshot for your reference.

Can you please let us know how can we update WordPress using Project Nami download page, as we don't have any sync button available in appservice on Azure under deployment options. Can we somehow do it manually using the zip file containing updates?

Any help in this regard would be appreciated.

Thank you.
![updates](https://user-images.githubusercontent.com/40587681/41888369-90a88c0e-791e-11e8-9a9d-7b412f7ffead.JPG)

**Comment by grzegorzszypa** [comment]
Hi the video is fine, but how to do it if I installed my wordpress from Azure Gallery, because in this way of deployment, this configuration option is missing?

**Comment by patrickebates** [comment]
To confirm, you don't see the following option in your app service?

![azuredeploy](https://user-images.githubusercontent.com/4357158/43039934-199dc10c-8d27-11e8-8f10-32f4d1d3b6a6.PNG)

**Comment by grzegorzszypa** [comment]
I see it but this option is not configured If I install it by Azure Portal

**Comment by patrickebates** [comment]
Configure it using the settings from the below the video in the post linked above.
![deploysettings](https://user-images.githubusercontent.com/4357158/43040118-0d5d611e-8d2b-11e8-95e6-e6edfb1c0ce1.PNG)

**Comment by mahfujenam** [comment]
Hi, please help to update my ProjectNami in local server or is it same as WordPress manual update! please help,

Thanks, 
Mahfuj

**Comment by patrickebates** [comment]
Yes, same as WordPress manual update.  Download the new release from GitHub and unpack over your existing installation.

**Comment by mahfujenam** [comment]
Many thanks Patrick,


My best,

Mahfuj


________________________________

**Comment by vijaykr0** [comment]
Why isn't the Azure portal set up to deploy the latest version of project nami in the first place? Why go through so many hoops?

**Comment by patrickebates** [comment]
At the time we were last able to update the version of PN hosted within the Azure library, it simply didn't support the commands we could use with the Deploy to Azure system.  On top of that, updating the deployment was quite a drill too.  They have since moved to a new deployment model within the past year, but we still haven't been able to work out the same ease of use that we have with Deploy to Azure yet.

For creating new sites on existing resources, I would recommend doing what I do myself.  Create a new Web App, then set the Deployment Options as listed above.  PN will be copied into the site and you can start the WP install as normal.
